### PR TITLE
Fix script/ install

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: dist/rpm/os-autoinst-openvswitch-test.spec [hunk 1]
```diff
@@ -19,7 +19,7 @@ touch %{_sourcedir}/%{short_name}
 %build
 # call one of the components but not openqa itself which would need a valid
 # configuration
-/usr/lib/os-autoinst/os-autoinst-openvswitch --help
+/usr/lib/os-autoinst/script/os-autoinst-openvswitch --help
```
Comment (kalikiana) [inline dist/rpm/os-autoinst-openvswitch-test.spec:22]: What's the context here? Was there a related change that broke tests? If not, how did you find out this was broken?

Comment (kalikiana) [reply dist/rpm/os-autoinst-openvswitch-test.spec:22]: I guess this might be related to #1978 - no CI failures there 🤔

Comment (okurz) [reply dist/rpm/os-autoinst-openvswitch-test.spec:22]: Yes,
* #1978

Comment (kalikiana) [reply dist/rpm/os-autoinst-openvswitch-test.spec:22]: Apparently I can't view OBS test results for that, or this PR.

**Pull request merged**